packages/web3-core/test/unit/web3_request_manager.test.ts

```diff
@@ -205,56 +199,6 @@ describe('Web3RequestManager', () => {
 			});
 		});
 
-		describe('ipc provider', () => {
```

Comment:
What do you think about moving this tests (or some of them), instead of deleting them, to somewhere inside: https://github.com/ChainSafe/web3.js/blob/726c9c365a83637316b613173b8c0783183f97fa/packages/web3-providers-ipc/test/unit/ipc_provider.test.ts

On the side but related: There is no test other than here that checks the validity of an ipc provider with a URL starting with `ipc://`. And I just discovered that we missed using some testing data for that and the file https://github.com/ChainSafe/web3.js/blob/0d9412dc605a2113af3c9d1a9e9dba7f0e792a6c/packages/web3-providers-ipc/test/fixtures/test_data.ts contains wrong data that supposed to be only for testing WebSocket and this file is not used. Would you update for this in this MR or should we create a new issue for this?

Reply:
> What do you think about moving this tests (or some of them), instead of deleting them, to somewhere inside: https://github.com/ChainSafe/web3.js/blob/726c9c365a83637316b613173b8c0783183f97fa/packages/web3-providers-ipc/test/unit/ipc_provider.test.ts

I can if you want me to but it's kinda odd for IPC package to have tests for the request manager. Imo, it would be a lot more beneficial to write (if there aren't already) web3-core tests that check if you can set any of the supported providers (separate issue maybe?)

> Would you update for this in this MR or should we create a new issue for this?

I'm gonna go on vacation next week, so it's probably best to create good first issue for that^^